lib/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -313,6 +313,11 @@ const rule = (primary, _secondaryOptions, context) => {
 				if (node.type !== 'function' || node.value.toLowerCase() !== 'calc') return;
 
 				const { nodes } = node;
+
+				if (!nodes.length) {
+					return;
+				}
```

Comment:
```suggestion
				if (!nodes.length) return;
```

Nit for a convention we're adopting for early returns as rules tend to have a few of them.